### ISSUE TITLE
sab 0.8.x -- config > servers - enable/disable wrong

### ISSUE THREAD

**Issue by thezoggy** [sabnzbd/sabnzbd#438]
on latest dev. when unchecking/checking servers.. if the sort order changes.. then the enable/disable is wrongly set. if the order doesnt change.. its fine. 

thus add two servers to sab with same priority..  uncheck the one that comes first.. watch it move to the last but the wrong is is disabled.. then when you renable it.. it swaps..

**Comment by thezoggy** [comment]
example of what happens when you have two servers at same priority, and you uncheck enable on the top one.. their places get swapped..and wrongly disabled:
![sab9](http://zoggy.net/sab9.png)

**Comment by thezoggy** [comment]
then had to clear browser cache, and it showed that both were really disabled now... and then trying to enable them is also funky.

**Comment by Safihre** [comment]
The problem is only partly SAB's, it's mostly Firefox that smartly tries to fill the forms for you on a page reload. I can reproduce it, but when you then look with Inspect Element, you can see only the ones that really should be checked have the ```checked="checked"``` part. If you simultaneously look at it in Chrome, you will see that there the page displays correctly.
Now I think the problem is that all these checkboxes have the same ```name``` and the only distinction is its ```rel```. Let's see if we can change that!

**Comment by thezoggy** [comment]
yeah.. prob bad html.. might wanna run each page through a validator... prob will find other reasons for 'bugs'

**Comment by Safihre** [comment]
I already verified the HTML of all Config page, and besides the use of ```rel``` they are all valid.

I modified the code to not use ```rel``` but ```name``` on the Server's page, but Firefox still did this stupid auto-filling.
But I found through: http://stackoverflow.com/questions/299811/why-does-the-checkbox-stay-checked-when-reloading-the-page
That it was just missing ```autcomplete="off"```, adding that fixes it.
